php/src/WindowAlertDemo/app.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$alert_closure = function() use ($window){
+    $window->alert('omg my first client side php') ;
```

Comment:
Lol, honestly, I've been writing PHP nearly a decade, and have been looking for something that can do this for ages... and it all works. I don't know where this has been all my life.